### ISSUE TITLE
PaddleSeg训练报错

### ISSUE THREAD

**Issue by xingyun8** [PaddlePaddle/PaddleSeg#3001]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

使用服务器上的这个预训练模型pretrained: https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz
训练显示The pretrained model directory is not Found: /home/qq/.paddleseg/pretrained_model/deeplab_mobilenet_fp32_fluid/model.pdparams

**Comment by xingyun8** [comment]
以下是我训练的yml文件
batch_size: 4
iters: 1000

train_dataset:
  type: OpticDiscSeg
  dataset_root: data/cc
  transforms:
    - type: Resize
      target_size: [513, 513]
    - type: RandomHorizontalFlip
    - type: Normalize
  mode: train

val_dataset:
  type: OpticDiscSeg
  dataset_root: data/cc
  transforms:
    - type: Resize
      target_size: [513, 513]
    - type: Normalize
  mode: val

optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  end_lr: 0
  power: 0.9

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

# distill_loss is used for distillation
distill_loss:
  types:
    - type: KLLoss
  coef: [3]

model:
  type: DeepLabV3P
  backbone:
    type: MobileNetV2_x1_0 
    pretrained: https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz

**Comment by juncaipeng** [comment]
估计是文件下载失败。
你可以手动下载 https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz  并解压到本地，然后在设置本地路径给 pretrained 字段。

**Comment by xingyun8** [comment]
> 估计是文件下载失败。 你可以手动下载https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz 并解压到本地，然后在设置本地路向预训练。

您好，我把这个文件手动下载解压后，里面也没有model.pdparams这个文件

**Comment by juncaipeng** [comment]
哦  这个预训练权重过时了，建议使用paddleseg和paddleclas提供的预训练下载权重。

**Comment by xingyun8** [comment]
好的，多谢。
